brew_view/controllers/system_list_api.py

```diff
@@ -42,20 +63,48 @@ def get(self):
         tags:
           - Systems
         """
-        self.logger.debug("Getting all Systems")
-
+        query_set = System.objects.order_by(self.request.headers.get('order_by', 'name'))
+        serialize_params = {'to_string': True, 'many': True}
+
+        include_fields = self.request.headers.get('include_fields')
+        exclude_fields = self.request.headers.get('exclude_fields')
+        include_commands = self.get_query_argument('include_commands', None)
+
+        if include_fields and exclude_fields:
+            raise ModelError("Headers 'include_fields' and 'exclude_fields' "
+                             "cannot exist on the same request")
+        elif include_commands and (include_fields or exclude_fields):
```

Comment:
I'm going to assume the reasoning for this is because it would be ambiguous if the include/excludes are referring to system or command attributes?

Reply:
Yeah, exactly. If I have `exclude_fields=commands` and `include_commands=true` there's no right answer.